packages/tags/package.json

```diff
@@ -74,8 +74,8 @@
         "@spectrum-web-components/shared": "^1.0.1"
     },
     "devDependencies": {
-        "@spectrum-css/tag": "^10.0.0-s2-foundations.15",
-        "@spectrum-css/taggroup": "^6.0.0-s2-foundations.15"
+        "@spectrum-css/tag": "10.0.0-s2-foundations.15",
```

Comment:
General question, as this seems to have changed on all components, why are we pinning the versions now?